src/context/wishListContext.js

```diff
@@ -1,26 +1,27 @@
 import { createContext, useContext, useReducer } from "react";
 import { privateInstance } from "../utils/axios";
 import { wishListReducer } from "../reducers";
+import { useToast } from "./toastContext";
 
 const WishListContext = createContext({wishList:[]})
 
 
 const WishListProvider = ({children}) => {
+    const {addToast} = useToast()
 
     const getWishListData = async () => {
-    console.log("i m getting triggered getWishList")
     try{
         const {status,data} = await privateInstance
       .get("/user/wishlist")
       if(status ===200){
         wishListDispatch({ type: "SET_DATA", payload: data.wishlist })
```

Comment:
Suggestion : here you can use payload: {wishlist : data.wishlist} that would be more understandable that which kind of data it is .

Reply:
Done